packages/node-runtime-worker-thread/src/child-process-evaluation-listener.ts

```diff
@@ -1,13 +1,15 @@
 import { ChildProcess } from 'child_process';
-import { exposeAll, WithClose } from './rpc';
+import { exposeAll, Exposed, close } from './rpc';
 import type { WorkerRuntime } from './index';
 import { RuntimeEvaluationListener } from '@mongosh/browser-runtime-core';
 
+type NoUndef<T> = { [k in keyof T]-?: T[k] };
```

Comment:
Is this [`Required<T>`](https://www.typescriptlang.org/docs/handbook/utility-types.html#requiredtype)?

Reply:
Omg, yes, thanks for spotting!